share/CMakeLists.txt

```diff
@@ -20,6 +20,13 @@ CONFIGURE_FILE(
     COPYONLY
     )
 
+EXECUTE_PROCESS(COMMAND python ${CMAKE_CURRENT_SOURCE_DIR}/dbtypes_gen.py ${core_version} OUTPUT_VARIABLE DBTYPES_JSON_ENTRIES)
```

Comment:
How should we determine that dbtypes_gen.py failed? The output variable will be empty (rightfully) if the version isn't passed correctly.